pom.xml

```diff
@@ -550,6 +550,12 @@
                 <version>${alfresco.glacier-connector.version}</version>
                 <type>amp</type>
             </dependency>
+            <dependency>
```

Comment:
@cagache ACS doesn't bring RM as a dependency by default. There is only a property added with the version, used for guidelines and for the integration testing.
So this dependency here needs to be removed.